### ISSUE TITLE
[Bug]: Wallpanel

### ISSUE THREAD

**Issue by w1tw0lf** [piitaya/lovelace-mushroom#1373]
### Requirements

- [X] I checked the troubleshooting section in the README to verify that I have the latest Mushroom version.
- [X] I did a search to see if there is a similar issue or if a pull request is open.
- [X] I have read the state of card mod support (in pin issues) and understand card mod support is not provided in this repository.

### Current Behavior

Using wallpanel(https://github.com/j-a-n/lovelace-wallpanel) to display info on a static sceen. When adding any type of mushroom card, the memory usage slowly increases and browser eventialy crashes, with a "aww snap" error.

### Expected Behavior

Used to work perfectly for the past year.

### Steps To Reproduce

RPi 4 loads chrome and displays a dashboard dedicated to wallpanel setup.

### Context

YAML state
```yaml

```

### Environment

```markdown
- Browser: chrome
- HA Version: 2024.2.1
- Mushroom: 3.4.2
```

### Anything else?

more info here as well: https://github.com/j-a-n/lovelace-wallpanel/issues/205
Setting up the same layout as on the wallpanel on normal dashboard, works perfectly.

**Comment by Pscharumbel** [comment]
Similar with me. I use Wallpanel with Mushroom Cards on an LG TV. After about 90 minutes, the browser crashes with the message 'Out of memory.' This wasn't a problem with the old version.

**Comment by w1tw0lf** [comment]
Just to add, if I configure it without any mushroom cards, it works perfectly.

**Comment by swampylee** [comment]
I have this too - latest Mushroom with latest HA and latest Wallpanel - refreshes are really slow, card updates significantly delayed and it seems directly related to Mushroom cards use.

**Comment by piitaya** [comment]
Can you share your dashboard configuration with Mushroom card?

**Comment by w1tw0lf** [comment]
I did this as a test, to see 
```
wallpanel:
  enabled: true
  hide_toolbar: true
  hide_sidebar: true
  fullscreen: true
  keep_screen_on_time: 0
  black_screen_after_time: 0
  image_url: media-entity://camera.google_photos_photo_frame_media
  show_image_info: false
  style:
    wallpanel-screensaver-info-box:
      '--wp-card-width': 457px
    wallpanel-screensaver-info-box-content:
      '--ha-card-background': none
      '--ha-card-box-shadow': none
      '--ha-card-border-width': 0px
      '--primary-text-color': '#e1e1e1'
      '--secondary-text-color': '#9b9b9b'
  cards:
    - entity: light.main_bedroom_lamp_left
      type: custom:mushroom-light-card
      layout: vertical
      name: Bedlamp Left    
      wp_style:
        grid-column: 1 / 1
        grid-row: 1 / 1
        background-color: '#00000099'
    - type: custom:button-card
      color_type: blank-card
      wp_style:
        grid-column: 2 / 2
        grid-row: 1 / 1
    - type: custom:button-card
      color_type: blank-card
      wp_style:
        grid-column: 3 / 3
        grid-row: 1 / 1
    - entity: light.main_bedroom_lamp_right
      type: custom:mushroom-light-card
      layout: vertical
      name: Bedlamp Left       
      wp_style:
        grid-column: 4 / 4
        grid-row: 1 / 1
        background-color: '#00000099'
views:
  - title: Home
    type: custom:grid-layout
    layout:
      gridrows: auto
      grid-template-columns: 25% 25% 25% 25%
      grid-template-areas: |
        "c1 c2 c3 c4"
    badges: []
    cards: []
```
If you keep an eye on either task manager in chrome or native you will see the memory slowly increase and the crashes chrome.

**Comment by swampylee** [comment]
> I have this too - latest Mushroom with latest HA and latest Wallpanel - refreshes are really slow, card updates significantly delayed and it seems directly related to Mushroom cards use.

I've been monitoring this today and while I do have a massive dashboard with many many mushroom cards I do see the RAM usage monitored through Task Manager just keep going up and up. It's not uncommon for me to see > 6GB allocated to that dashboard tab in Chrome.  As such, the page becomes unresponsive after a while. I also tested in Edge and see the same behaviour. I'm not convinced it's Wallpanel that's the problem as I've been using that for multiple dashboards for a long time, I think it might be related to the changes made to Mushroom. I did notice that if I go back to Mushroom 3.3.0 the problem seems to disappear.

**Comment by diemon24** [comment]
Yes it's true. I have this problem on Wallpanel in any browsers.  Maybe I can install old ver cards?

**Comment by diemon24** [comment]
No, this is not a solution to the error. The old version behaves the same way - the browser gives an out of memory error.

**Comment by diemon24** [comment]
> I've been monitoring this today and while I do have a massive dashboard with many many mushroom cards I do see the RAM usage monitored through Task Manager just keep going up and up. It's not uncommon for me to see > 6GB allocated to that dashboard tab in Chrome. As such, the page becomes unresponsive after a while. I also tested in Edge and see the same behaviour. I'm not convinced it's Wallpanel that's the problem as I've been using that for multiple dashboards for a long time, I think it might be related to the changes made to Mushroom. I did notice that if I go back to Mushroom 3.3.0 the problem seems to disappear.

I tested 3.3.0, but problem remains... 
I don't know what's happened... 

I used wallpanel with mushroom cards for a long time and it worked. But now I'm installing old versions of cards in hucs (without uninstalling) and it doesn't work correctly.

**Comment by Pscharumbel** [comment]
Maybe HA 2024.2 is the cause with the new Python version?

**Comment by Pscharumbel** [comment]
HAOS

**Comment by diemon24** [comment]
> HAOS

I updated python ver in Synology OS to 3.11. Testing. But I think, docker container have last python ver

**Comment by SmartHomeFanatic** [comment]
I already removed Wallpanel but still have the same behaviour. Started with 2024.2

**Comment by Pscharumbel** [comment]
I've reverted back to card-mod 3.4.0. Everything is running smoothly again.

**Comment by piitaya** [comment]
So card mod was the issue? I can not reproduce the bug you mentioned with Mushroom cards only. Does someone have the issue **without** card mod installed?

**Comment by diemon24** [comment]
> So card mod was the issue? I can not reproduce the bug you mentioned with Mushroom cards only. Does someone have the issue **without** card mod installed?

Yes, I using card mod. You think this is problem?

**Comment by SmartHomeFanatic** [comment]
Thanks so much. I was searching for the issue for weeks now. My whole frontend was unacceptable slow. It was driving me mad. I already removed lots of stuff and moved all tabs of my dashboard into a new one for testing etc.

Firefox tabs crashed when having them open for a longer time. It really was freaking me out.

Today I removed card-mod and everything is fast again.... thank you so much for bringing my attention to that.
Really wonder how this could influence the performance of the whole system that much.

**Comment by w1tw0lf** [comment]
I see the opposite... have wallpanel running without mushroom cards and card mods, it has been stable without any issues.

Running wallpanel with mushroom cards, causes the browser to crash. I have tried it with just 1 or 2 basic mushroom light cards and crashes.

**Comment by piitaya** [comment]
@w1tw0lf Without card mod installed?

**Comment by w1tw0lf** [comment]
@piitaya with card mod install.

**Comment by piitaya** [comment]
Try uninstalling card mod and see what happens, or at least, try to rollback to a previous card mod version (3.4.0).

**Comment by w1tw0lf** [comment]
Sorry... it works perfectly with card mod.

When I add mushroom cards it causes the browser to crash.

**Comment by piitaya** [comment]
And does it works with mushroom card without card mod? Maybe there is an incompatibility between those two.

**Comment by diemon24** [comment]
> I've reverted back to card-mod 3.4.0. Everything is running smoothly again.

I returned old version card mod 3.4.0. I need time for test this

**Comment by diemon24** [comment]
> > I've reverted back to card-mod 3.4.0. Everything is running smoothly again.
> 
> I returned old version card mod 3.4.0. I need time for test this 

I set up automation in XA to reload the browser when it reaches 55% memory. At 20:05 today I installed the old version of card mod and manually restarted the browser. Now I see that there are some minor changes in the memory consumption graph. but this is not the final test

![image](https://github.com/piitaya/lovelace-mushroom/assets/91910101/ef6c21ac-78ca-4cd2-8ccc-24c266ce6555)
